DiceKitTests/MultiplicationExpression_Tests.swift

```diff
@@ -113,9 +113,67 @@ extension MultiplicationExpression_Tests {
         expect(result.multiplierResult) == expectedMultiplierResult
         expect(result.multiplicandResults) == expectedMultiplicandResults
     }
+
+}
+
+// MARK: - Operators
+extension MultiplicationExpression_Tests {
+    
+    /// Same as test_evaluate_shouldCreateResultCorrectly(),
+    /// but using an operator overload instead of MultiplicationExpression
+    func test_operator_shouldCreateMultiplicationExpression() {
```

Comment:
We don't need to test the results, only that the operator creates the expression correctly.